warehouse/templates/packaging/detail.html

```diff
@@ -45,9 +45,9 @@
   {% set icon = "fab fa-reddit-alien" %}
 {% elif name.lower().startswith("slack") or parsed.netloc == "slack.com" or parsed.netloc.endswith(".slack.com") %}
   {% set icon = "fab fa-slack" %}
-{% elif parsed.netloc == "twitter.com" or parsed.netloc.endswith(".twitter.com") %}
+{% elif parsed.netloc in ["twitter.com", "x.com"] or parsed.netloc.endswith((".twitter.com", "x.com")) %}
```

Comment:
```suggestion
{% elif parsed.netloc in ["twitter.com", "x.com"] or parsed.netloc.endswith((".twitter.com", ".x.com")) %}
```

otherwise `timex.com` would match